app/helpers/user_helper.rb

```diff
@@ -74,7 +74,8 @@ def auth_button_preferred(name, provider, options = {})
       image_tag("#{name}.svg",
                 :alt => t("application.auth_providers.#{name}.alt"),
                 :class => "rounded-1 me-3",
-                :size => "24") + t("application.auth_providers.#{name}.title"),
+                :width => "24px",
+                :height => "24px") + t("application.auth_providers.#{name}.title"),
```

Comment:
This was only just changed in b4f719a8c58c5177fe3dbc330fc90b3e0ef94c4c to make it consistent with the `auth_button` method so why are we changing it back again?

Reply:
Ah yes, I changed it back.